tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -69,6 +69,7 @@
 # Visualize averaged sensor space data
 evoked = epochs.average()
 evoked.plot_joint()
+evoked.crop(0.05, 0.15, verbose='error')  # ignore baseline
```

Comment:
I think you should move this down into the next section, as it could be confusing having it up here, under "Visualize data"